specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json

```diff
@@ -779,6 +779,18 @@
             "type": "string"
           },
           "type": "object"
+        },
+        "managedVirtualNetwork": {
+          "description": "Workspace managed virtual network",
+          "type": "string"
+        },
+        "privateEndpointConnections": {
+          "description": "Private endpoint connections to the workspace",
+          "allOf": [
+            {
+              "$ref": "./privateEndpointConnections.json#/definitions/PrivateEndpointConnectionList"
+            }
+          ]
```

Comment:
this should be an array of PrivateEndpointConnections, not the List response

Reply:
> this should be an array of PrivateEndpointConnections, not the List response

fixed